javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,103 @@
+// Flight booking fullname function
+
+console.log(getFullName("khuram", "Ejaz", true));
+console.log(getFullName("john", "smith"));
+
+function getFullName(firstName1, lastName1, useFormalName) {
+  if (useFormalName == true && firstName1 !== "" && lastName1 !== "") {
+    return "Lord " + firstName1 + " " + lastName1;
+  } else if (
+    useFormalName == false &&
+    firstName1 !== null &&
+    lastName1 !== null
+  ) {
+    return firstName1 + " " + lastName1;
+  } else if (
+    useFormalName == null &&
+    firstName1 !== null &&
+    lastName1 !== null
+  ) {
+    return firstName1 + " " + lastName1;
+  } else {
+    console.log("Please enter the first name and last name");
+  }
+}
+
+// Event application
+
+function getEventWeekDay(x) {
+  let d = new Date();
+  let day = (d.getDay() + x) % 7;
+  var days = [
```

Comment:
Var is not used as per ES6 specifications, so use const or let keyword based on variable requirements.